CMakeLists.txt

```diff
@@ -7,656 +7,75 @@ set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 include(ExternalProject)
 include(GNUInstallDirs)
 
-# Define Options
-option(BUILD_PYTHON "Include Python" ON)
-option(BUILD_QT "Include Qt" ON)
-option(BUILD_SIP "Include SIP" ON)
-option(BUILD_PYQT "Include PyQt" ON)
-option(BUILD_NUMPY "Include Numpy" ON)
-option(BUILD_SCIPY "Include Scipy" ON)
-option(BUILD_PYSERIAL "Include PySerial" ON)
-option(BUILD_NUMPY_STL "Include Numpy-STL" ON)
-option(BUILD_ZEROCONF "Include python-zeroconf" ON)
+# find_package(CuraBuildEnvironment)
```

Comment:
I want to include a CMake config file in CuraBuildEnvironment that indicates it is available and provides access to the three CMake scripts that are included below.